src/main/java/com/minecolonies/util/AchievementUtils.java

```diff
@@ -0,0 +1,29 @@
+package com.minecolonies.util;
+
+import com.minecolonies.colony.Colony;
+import com.minecolonies.colony.permissions.Permissions;
+import net.minecraft.stats.Achievement;
+
+import java.util.List;
+
+/**
+ *
+ */
+public final class AchievementUtils {
+
+    public static void syncAchievements(final Colony colony) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)